docs/monitoring/model-monitoring.md

```diff
@@ -0,0 +1,84 @@
+The model server is then wrapped around a machine learning model that uses a function to calculate predictions based on the available vectors. Next, the model 
+server creates a log for the input and output of the vectors, and the entries are written to the production data stream (a v3io stream). While the model server 
+is processing the vectors, the stream function monitors the log of the data stream and is triggered when a new log entry is detected. The stream function examines 
+the log entry, processes it into statistics which are then written to the statistics databases (parquet file, time series database and key value database). 
+The parquet files are written as a feature set under the model monitoring project. The parquet files can be read either using `pandas.read_parquet` or `feature_set.get_offline_features`, 
+like any other feature set. In parallel, a scheduled MLRun job runs reading the parquet files, performing drift analysis. The drift analysis data is stored so 
+that the user can retrieve it in the Iguazio UI or in a Grafana dashboard.
+
+When you enable model monitoring, you effectively deploy three components:
+- application controller function: handles the monitoring processing and the triggers the apps that trigger the writer. The frequency is determined by `base_period`. 
+- stream function: monitors the log of the data stream. It is triggered when a new log entry is detected. The monitored data is used to create real-time dashboards, detect drift, and analyze performance.
+- writer function: writes to the database and outputs alerts.
+
+## APIs
+
+The model monitoring APIs are configured per project. The APIs are:
+
+- {py:meth}`~mlrun.db.httpdb.HTTPRunDB.enable_model_monitoring` &mdash; brings up the controller and schedules it according to the `base_period`; deploys the writer.
+- {py:meth}`~mlrun.projects.MlrunProject.disable_model_monitoring` &mdash; disables the controller. 
+- {py:meth}`~mlrun.projects.MlrunProject.set_model_monitoring_function` &mdash; Update or set a monitoring function to the project. (Monitoring does not start until the function is deployed.) 
+- {py:meth}`~mlrun.projects.MlrunProject.create_model_monitoring_function` &mdash; creates a function but does not set it. It's useful for troubleshooting, since it does  not register the function to the project.
+- {py:meth}`~mlrun.projects.MlrunProject.list_model_monitoring_functions` &mdash; Retrieves a list of all the model monitoring functions.
+- {py:meth}`~mlrun.projects.MlrunProject.remove_model_monitoring_function` &mdash; Removes the specified model-monitoring-app function from the project and from the DB.
+- {py:meth}`~mlrun.projects.MlrunProject.set_model_monitoring_credentials` &mdash; Sets the Kafka or SQL credentials to be used by the project's model monitoring infrastructure functions. 
+
+## Configuration flow
+
```

Comment:
I think code blocks are needed here, not just text

Reply:
added